include/fw_info.h

```diff
@@ -21,45 +21,7 @@ extern "C" {
 #if USE_PARTITION_MANAGER
 #include <pm_config.h>
 #endif
-
-/** @defgroup fw_info Firmware info structure
- * @{
- */
-
-#define MAGIC_LEN_WORDS (CONFIG_FW_INFO_MAGIC_LEN / sizeof(uint32_t))
-
-
-/**
- * This struct serves as a header before an EXT_API.
- *
- * @details This header contains basic metadata about the EXT_API: A unique
- *          identifier, version and flags to be used for compatibility matching.
- *          The payload that comes after this header is completely
- *          implementation dependent for each EXT_API.
- *
- * @note How to use the EXT_API, such as the signatures of all the functions in
- *       the list must be unambiguous for an ID/version combination.
- */
-struct __packed fw_info_ext_api {
-	/* Magic value to verify that the struct has the correct format.
-	 * The magic value will change whenever the format changes.
-	 */
-	uint32_t magic[MAGIC_LEN_WORDS];
-
-	/* The length of this header plus everything after this header. Must be
-	 * word-aligned.
-	 */
-	uint32_t ext_api_len;
-
-	/* The id of the EXT_API. */
-	uint32_t ext_api_id;
-
-	/* Flags specifying properties of the EXT_API. */
-	uint32_t ext_api_flags;
-
-	/* The version of this EXT_API. */
-	uint32_t ext_api_version;
-};
+#include <fw_info_struct.h>
```

Comment:
Is this name a bit misleading? Its more than just the struct right?  `fw_info_bare.h`